pandas/tests/series/test_operators.py

```diff
@@ -1857,3 +1857,15 @@ def test_op_duplicate_index(self):
         result = s1 + s2
         expected = pd.Series([11, 12, np.nan], index=[1, 1, 2])
         assert_series_equal(result, expected)
+
+    def test_argminmax(self):
+        # GH13595
+
+        # Expected behavior for arg min/max in the presence of NA and Inf
+        s = pd.Series([0, -np.inf, np.inf, np.nan])
+
+        assert s.argmin() == 1
+        assert np.isnan(s.argmin(skipna=False))
+
```

Comment:
if you do

```
with pd.option_context('mode.use_inf_as_null', True):
    ...
```

I suspect this will fail. Can you test with this as well.
(you may have to get the value of the option and pass to ``isfinite=pd.get_option('mode.use_inf_as_null')``)